website/source/docs/guides/kuberenetes-deployment.html.md

```diff
@@ -0,0 +1,177 @@
+
+Now, you can deploy Consul using `helm install`. This will deploy three servers
+and agents on all Kubernetes nodes. The process should be quick, less than 5
+minutes.  
+
+```sh 
+$ helm install ./consul-helm
+
+NAME:   mollified-robin LAST DEPLOYED: Mon Feb 25 15:57:18 2019 NAMESPACE: default STATUS: DEPLOYED
+NAME                             READY  STATUS             RESTARTS  AGE
+mollified-robin-consul-25r6z     0/1    ContainerCreating  0         0s
+mollified-robin-consul-4p6hr     0/1    ContainerCreating  0         0s
+mollified-robin-consul-n82j6     0/1    ContainerCreating  0         0s
+mollified-robin-consul-server-0  0/1    Pending            0         0s
+mollified-robin-consul-server-1  0/1    Pending            0         0s
+mollified-robin-consul-server-2  0/1    Pending            0         0s
+```
+
+The output above has been reduced for readability. However, you can see that
+there are three Consul servers and three Consul clients on this three node
+Kubernetes cluster. 
+
+## Access Consul UI
+
+To access the UI you will need to update the `ui` values in the Helm chart.
+Alternatively, if you do not wish to upgrade your cluster, you can set up [port
+forwarding]
+(https://www.consul.io/docs/platform/k8s/run.html#viewing-the-consul-ui) with
+`kubectl`. 
+
+### Create Values File
+
+First, create a values file that can be passed on the command line when
+upgrading.
+
+```yaml
+# values.yaml
+global: 
+  datacenter: hashidc1 
+syncCatalog: 
+  enabled: true 
+ui: 
+  service: 
+    type: "LoadBalancer" 
+```
+
+This file renames your datacenter, enables catalog sync, and sets up a load
+balancer service for the UI. The catalog sync parameters will allow you to see
+the Kubernetes services in the Consul UI. 
+
+### Initiate Rolling Upgrade 
+
+Finally, initiate the
+[upgrade](https://www.consul.io/docs/platform/k8s/run.html#upgrading-consul-on-kubernetes)
+with `helm upgrade` and the `-f` flag that passes in your new values file. This
+processes should also be quick, less than a minute.  
+
+```sh
+$ helm upgrade consul -f values.yaml 
+```
+
+You can now use `kubectl` to discover the external IP of your Consul UI.
+
+```sh 
+$ kubectl get services 
+NAME                            TYPE           CLUSTER-IP     EXTERNAL-IP             PORT(S)        AGE 
+consul                          ExternalName   <none>         consul.service.consul   <none>         11d 
+kubernetes                      ClusterIP      122.16.14.1    <none>                  443/TCP        137d
+mollified-robin-consul-dns      ClusterIP      122.16.14.25   <none>                  53/TCP,53/UDP  13d
+mollified-robin-consul-server   ClusterIP      None           <none>                  8500/TCP       13d
+mollified-robin-consul-ui       LoadBalancer   122.16.31.395  36.276.67.195           80:32718/TCP   13d
+```
+
+The service should have `consul-ui` appended to the deployment name. Note, you
+do not need to specify a port when accessing the dashboard. 
+
```

Comment:
I think adding a description and output for the `syncCatalog` being enabled using the above values file should be showcased here as well.

Reply:
Let me try to make some updates and see if it still flows naturally. Did you have any particular example in mind? I was thinking of changing `consul members` below to `consul catalog services`. That way I can point out that some of the service are actually synced from k8s.

Reply:
I would just show the catalog sync pods spun up after opting into `syncCatalog`. This will let people know that we do have another process that takes care of syncing the service catalog.